guides/WINDOWS_SETUP.md

```diff
@@ -0,0 +1,61 @@
+# Windows Development Setup Guide for WrongSecrets CTF Party
+
+This guide provides a step-by-step process for setting up the local development environment for WrongSecrets CTF Party on a Windows machine.
+
+## 1. Prerequisites
+
+Before you begin, you must install the following tools:
+- **Git:** [https://git-scm.com/downloads](https://git-scm.com/downloads) (Ensure **Git Bash** is installed)
+- **Docker Desktop for Windows:** [https://www.docker.com/products/docker-desktop/](https://www.docker.com/products/docker-desktop/)
+- **Minikube:** [https://minikube.sigs.k8s.io/docs/start/](https://minikube.sigs.k8s.io/docs/start/)
+- **Helm:** [https://helm.sh/docs/intro/install/](https://helm.sh/docs/intro/install/)
+- **kubectl:** (Included with Docker Desktop)
+- **yq:** [https://github.com/mikefarah/yq/#install](https://github.com/mikefarah/yq/#install)
+
+## 2. System Configuration
+
+These Windows-specific configurations are crucial for a successful setup.
+
+### a. Enable Hardware Virtualization (BIOS/UEFI)
+
+Minikube and Docker require hardware virtualization (VT-x or AMD-V) to be enabled in your computer's BIOS/UEFI. You will need to restart your computer and press a key (like `F2`, `F10`, or `DEL`) during boot to enter the setup menu. Search online for instructions specific to your computer model.
+
+### b. Configure Docker Desktop Memory (WSL 2)
+
+The project requires at least 11GB of RAM. You must configure Docker's WSL 2 backend to provide this memory.
```

Comment:
this is only in the default configuraiton. Feel free to suggest users to change the confguration of the startup by using 8 GB instead for instance.